src/Shared/SFA.DAS.ProviderPayments.Calc.Shared.Infrastructure/Data/Repositories/DasAccountRepository.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+using SFA.DAS.Payments.DCFS.Infrastructure.Data;
+using SFA.DAS.ProviderPayments.Calc.Shared.Infrastructure.Data.Entities;
+
+namespace SFA.DAS.ProviderPayments.Calc.Shared.Infrastructure.Data.Repositories
+{
+    public class DasAccountRepository : DcfsRepository, IDasAccountRepository
+    {
+        public DasAccountRepository(string transientConnectionString) 
+            : base(transientConnectionString) { }
+
+        public void AddMany(List<DasAccountEntity> dasAccounts)
+        {
+            ExecuteBatch(dasAccounts.ToArray(), "Reference.DasAccounts");
+        }
+
+        public void UpdateBalance(long accountId, decimal balance)
```

Comment:
Should be increment balance. We are adding the balance to the existing balance

Reply:
will do